server/tsconfig.json

```diff
@@ -12,7 +12,10 @@
     "outDir": "build",
     "skipLibCheck": true,
     "strict": true,
-    "baseUrl": "./src/"
+    "baseUrl": "./src/",
+    "typeRoots": [
+      "./custom_typings"
```

Comment:
Should we be consistent and use `-` case for file and folder names? :p

Reply:
I was following the node_modules cos this one is the same level as that. But I guess we can just standardise all the files that we make.

Reply:
Also, I realise I can put this folder inside src then it'll be better.